pkg/volume/util/types/types.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package types defines types used only by volume componenets
+package types
+
+import "k8s.io/kubernetes/pkg/types"
+
+// UniquePodName defines the type to key pods off of
+type UniquePodName types.UID
```

Comment:
It probably would be. I see most people just use `types.UID`. We can move it in the future if we want to.